home/templates/home/about_page.html

```diff
@@ -74,7 +74,7 @@ <h4 class="no-margin">Angie Valderrama</h4>
 		<div class="sobre-nosotros__texto">
 			<div class="entrada__imagen">
 				<picture>
-					<img class="circular--square" src="{% static 'img/ana.png' %}" width="300" height="300" alt="img Team">
+					<img class="circular--square" src="{% static 'img/ana.png' %}" width="150" height="150" alt="img Team">
```

Comment:
May be you move these stylings like `width` and `height`  to a css class which then you can reuse